internal/integrations/model.go

```diff
@@ -0,0 +1,72 @@
+package integrations
+
+import "time"
+
+// This code is taken directly from the infrastructure agent
+// To avoid an extra dependency it has been copied/pasted
```

Comment:
Can we link the exact commit where this comes from? That way we should get some sort of traceability.

Reply:
I will add it, unfortunately, there is not a single commit for that config, since it has been modified several times in time